src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -103,6 +126,26 @@ private bool SetFeePerByte(ApplicationEngine engine, long value)
             return true;
         }
 
+        [ContractMethod(0_03000000, CallFlags.WriteStates)]
+        private bool SetExecFeeFactor(ApplicationEngine engine, uint value)
+        {
+            if (value == 0 || value > MaxExecFeeFactor) throw new ArgumentOutOfRangeException(nameof(value));
```

Comment:
Maybe if we allow 0 (checking the magic), we can configure a free private network easier.

Reply:
It is too dangerous. Maybe a minimum value should be added to protocol.json.

Reply:
This certainly can be done later if needed.